stankove_napady.md

```diff
@@ -0,0 +1,105 @@
+Zkusim popsat svuj navrh a zkusme to spolecnymi pripominkami upravit do lepsi podoby. Je to moje stankova premiera.
+<br>
+</br>
+## Co chceme prezentovat?
+
+**python.cz** 
+
+> aby je navstevovalo vice pythonistu i zacatecniku zajemcu o python
+
+> aby vedeli jake informace tam najdou
+
+**pyvo.cz**
+
+> protoze lide maji vedet, ze kazda zasadni zmena okolo mista ci casu srazu by se mela jako prvni objevit predevsim tam? tzn. naucit lidi hledat ze vsech kanalu dulezito info prave tady
+
+**PyLadies a Advanced srazy**
+
+> zvysit povedomi o PyLadies celkove (stale se setkavam jako organizatorka s tim, ze spousty python lidi o teto aktivite lidi z komunity vubec nevi)
+
+> zvysit povedomi i u mensich firem, ktere by to jakkoliv rady podporily
+
+> moznost jak zapojit zase nove kouce, kteri by napriklad opravdu radi Python ucili o moznosti na PyLadies nevi
```

Comment:
👍 

chtelo by to i nejak zpropagovat ty Ucime Python skupiny, neni na ne odkaz ani z python.cz jeste a ani obrazek jsem jim nenakreslil :( jsou takovy nedodelany